internal/backends/gl/event_loop.rs

```diff
@@ -371,7 +379,6 @@ fn process_window_event(
         WindowEvent::Resized(size) => {
             let size = size.to_logical(runtime_window.scale_factor() as f64);
             runtime_window.set_window_item_geometry(size.width, size.height);
-            window.set_size(size);
```

Comment:
This was the key to solve the jitter:  We need to keep the size from the resize event, IIRC